solidity/deploy/02_deploy_vending_machine.ts

```diff
@@ -0,0 +1,28 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCToken = await deployments.get("TBTCToken")
+  const TBTCTokenV2 = await deployments.get("TBTCTokenV2")
+
+  const unmintFee = 1000000000000000 // 0.001 of the amount being unminted
+
+  // This deployment is named VendingMachineV2 to avoid conflict with
+  // VendingMachine deployment which represents the vending machine for tBTC v1
+  // shipped via an external dependency.
+  await deploy("VendingMachineV2", {
```

Comment:
This is related to https://github.com/keep-network/tbtc-v2/pull/17/files#r682670289 and a previous comment about distinguishing `TBTCToken` from V1 and V2. Maybe we should rename the pulled `VendingMachine.json` as well?

Reply:
For anything tBTC v2, let's skip the suffix.

Reply:
If we skip the suffix, the `VendingMachine` artifact for tBTC v2 will shadow the `VeningMachine` artifact for v1 in the resulting package. Are we ok with that? Shouldn't we differentiate them somehow?

Reply:
I don't think it will be a problem if we shadow V1's `VendingMachine` as migrated V1 and V2 will be separate npm packages. In the package, we are publishing here we want to have V2 contracts, we don't care about external dependencies as those should be pulled from other packages if needed.

Reply:
Ok then: https://github.com/keep-network/tbtc-v2/pull/17/commits/c531f47f6931afcd0aaf664f3806608f1a284d47